docs/howtos/multipleflows.md

```diff
@@ -0,0 +1,24 @@
+# MultipleFlows
+
+
+The user can define a list of flows and their projects (list of lists) that are to be launched in the order set by him. Flowys must be registered with the prefect. If one of the subflows fails, the next ones fail and the main flow status will fail.
```

Comment:
Typo: "Flowys" -> "Flows".
Grammar: "set by him" -> "set by them".
Logic: "main flow status will fail" -> "main flow will fail" or "main flow status will be set to "failed""